rebasehelper/cli.py

```diff
@@ -49,12 +49,51 @@ def _format_action_invocation(self, action):
         return text.replace(' BUILDER_OPTIONS', '=BUILDER_OPTIONS')
 
     def _expand_help(self, action):
+        if hasattr(action, "actual_default"):
+            action.default = action.actual_default
```

Comment:
You could just use:
```python
action.default = getattr(action, 'actual_default', None)
```

Also, it would be nice to honor quoting style, i.e. use single quotes like everywhere else in this class.